src/main/java/core/basesyntax/RandomColors.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum RandomColors {
```

Comment:
```suggestion
public enum Colors {
```

мне кажется, это более подходящее название